main/src/ca/uwaterloo/flix/tools/pkg/Dependency.scala

```diff
@@ -29,4 +31,6 @@ object Dependency {
 
   case class MavenDependency(groupId: String, artifactId: String, version: SemVer, kind: DependencyKind) extends Dependency
 
+  case class JarDependency(url: URL, website: String, fileName: String) extends Dependency
```

Comment:
website?

Reply:
If we are fetching "google.com/coolJar.jar" I put the downloadet jar in the folder "google.com", so I need the name of the site saved to avoid a bunch of string manipulations later. I could also call it "folderName" to match "fileName"?

Reply:
Ah, I see. But I suppose a URL might not have a domain? (if it is an IP)

Reply:
I don't think we can do this.

The URL could also be a local path `C:\...`.

I suggest that we have to put the jar directly into the lib folder.